src/renderer/qssWidget/js/qssOpenUsbWidget.js

```diff
@@ -0,0 +1,350 @@
+     * the widget with a delay.
+     * @param {KeyboardEvent} keyboardEvent - The keyboard event (if any) that led to the
+     * change in the setting's value.
+     */
+    gpii.qssWidget.openUSB.close = function (that, closeTimer, keyboardEvent) {
+        that.closingKeyboardEvent = keyboardEvent;
+        closeTimer.start(that.options.closeDelay);
+    };
+
+    /**
+     * Adds a listener which stops the close timer in case the dialog was closed before
+     * the timer has finished as a result of another user interaction (e.g. clicking
+     * outside of the widget's window or clicking on a different QSS button which opens
+     * the QSS widget again).
+     * @param {Component} closeTimer - An instance of `gpii.app.timer` used for closing
+     * the widget with a delay.
+     */
+    gpii.qssWidget.openUSB.addVisibilityChangeListener = function (closeTimer) {
+        $(document).on("visibilityChange.qssOpenUSBWidget", function () {
+            if (document.visibilityState === "hidden") {
+                closeTimer.clear();
+            }
+        });
+    };
+
+    /**
+     * Removes the listener for clearing the `closeTimer`. Useful when the component is
+     * destroyed.
+     */
+    gpii.qssWidget.openUSB.removeVisibilityChangeListener = function () {
+        $(document).off("visibilityChange.qssOpenUSBWidget");
+    };
+
+    /**
+     * Creates an array of possible values for the given setting to be provided to the
+     * `gpii.psp.repeater` instance.
+     * @param {Object} setting - the current setting for the QSS menu.
+     * @return {Object[]} - An array of key-value pairs describing the key and the value
+     * (the visible name) of the setting.
+     */
+    gpii.qssWidget.openUSB.getRepeaterItems = function (setting) {
+        var schema = setting.schema || {},
+            values = schema["enum"],
+            keys = schema.keys || values;
+
+        return fluid.transform(keys, function (key, index) {
+            return {
+                key: key,
+                value: values[index]
+            };
+        });
+    };
+
+    /**
+     * Gets the value the pressed button from the menu and determines which even to fire
+     * in case of no valid button type is pressed does nothing
+     *
+     * @param {String} value - accepts only "Mount" and "Unmount"
+     * @param {Component} openUSB - The `gpii.qssWidget.openUSB` instance
+     * @param {KeyboardEvent} keyboardEvent - The keyboard event (if any) that led to the
+     * change in the setting's value.
+     * @param {EventListener} mountUsbEvent - handle to the onQssOpenUsbRequested event
+     * @param {EventListener} unmountUsbEvent - handle to the onQssUnmountUsbRequested event
+     * @param {String} messageChannel - The channel to which the message should be sent.
+     */
+    gpii.qssWidget.openUSB.handleOpenUSB = function (value, openUSB, keyboardEvent, mountUsbEvent, unmountUsbEvent, messageChannel) {
+        if (fluid.isValue(value)) {
```

Comment:
No particular need for a switch for a 2-element choice - also no point in failing silently since this represents a design error.
How about
if (value === "Mount") {
 ....
} else if (value === "Unmount") {
  ....
} else {
    fluid.fail("Unrecognised command to handleOpenUSB: " + value)
}